requirements.txt

```diff
@@ -28,3 +28,7 @@ aioblescan      # Replacement for beacontools for Tilt support
 pyzmq<17
 tornado<5
 
+# Django Rest Framework and related
+djangorestframework
+djangorestframework-jwt
```

Comment:
I'll need to do some research on these two packages, as I haven't used them before. What does JWT do?